tests/foreman/cli/test_repository.py

```diff
@@ -560,6 +562,34 @@ def test_positive_create_docker_repo_with_name(self, repo_options, repo):
         for key in 'url', 'content-type', 'name':
             assert repo.get(key) == repo_options[key]
 
+    @pytest.mark.tier1
+    def test_positive_create_with_new_organization_and_location(self):
+        """Check if repo can be created with a new Organization and Location.
+
+        :parametrized: no
+
+        :expectedresults: Repository is successrully created.
+
+        :BZ: 1992967
+
+        "CaseImportance: Critical
+        """
+        new_org = make_org()
+        new_location = make_location()
+        new_product = make_product(
+            {'organization-id': new_org['id'], 'description': 'test_product'}
+        )
+        Org.add_location({'location-id': new_location['id'], 'name': new_org['name']})
```

Comment:
Should we assign this to a variable and then assert on it below instead of a direct call ?

Reply:
Actually, should be fine